silk/node/wpantund_base.py

```diff
@@ -115,9 +115,9 @@ def un_whitelist_node(self, node):
         """
         self.remove(wpan.WPAN_MAC_WHITELIST_ENTRIES, node.get(wpan.WPAN_EXT_ADDRESS)[1:-1])
```

Comment:
Please modify whitelist and blacklist to Allowlist and DenyList here and in wpan_constants.py also to be in sync with the latest ot commit 
f2e15018f9b362b37f4c079634636a99e4502300 (HEAD -> master, origin/master, origin/HEAD)
Author: Jonathan Hui <jonhui@google.com>
Date:   Fri Sep 11 16:37:26 2020 -0700

    [mac-filter] change whitelist/blacklist to allowlist/denylist (#474)
    
    To promote inclusivity and eliminate any unintentional bias.

Reply:
I created a new issue #63 for this. I can address this in a new PR next week.